### PR TITLE
Added an event to Kentucky

### PR DESCRIPTION
**Pull request by Abhishek-NK**
"Police dogpiling a protestor and then targets a witness | June 16th" added into Kentucky.

### CODE REVIEW

File: reports/Kentucky.md [hunk 1]
```diff
@@ -119,3 +119,15 @@ id: ky-louisville-7
 **Links**
 
 * [Video taken from across the street](https://www.reddit.com/r/Louisville/comments/gw5joa/peaceful_protesters_marching_in_st_mathews_get/)
+
+### Police dogpiling a protestor and then targets a witness | June 16th
```
Comment (emarcey) [inline reports/Kentucky.md:123]: I think your subject-verb agreement is off here. Maybe "Police tackle protestor, then target witness"

File: reports/Kentucky.md [hunk 2]
```diff
@@ -119,3 +119,15 @@ id: ky-louisville-7
...
+
+A guy records a half-dozen policemen dogpiling a protestor and then one of them shoots the window to stop him from recording.
+
+tags: rubber-bullets, shoot
```
Comment (emarcey) [inline reports/Kentucky.md:127]: Can you update the tags to include attacking the protestor? I would suggest `tackle, beat, baton`

File: reports/Kentucky.md (hunk 1 above)
Comment (Abhishek-NK) [reply reports/Kentucky.md]: Thanks, I've corrected it.

File: reports/Kentucky.md (hunk 2 above)
Comment (Abhishek-NK) [reply reports/Kentucky.md]: I also added these tags.

**Pull request merged**